testing/_test_suite.ts

```diff
@@ -383,5 +389,11 @@ export class TestSuiteInternal<T> implements TestSuite<T> {
     } else {
       await fn.call(context, t);
     }
+
+    if (assertionState.checkAssertionErrorStateAndReset()) {
```

Comment:
I feel this check is unnecessary. If I removed this check, the test cases still pass.

I'm removing this block.

Reply:
There is a little tricky about this, I think you run the test inside `Deno.test` and then it will never throw error. But if you remove it, then it works.